README.md

```diff
@@ -55,3 +55,7 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+###TESTING
+
+Run all from the testing tab lol
```

Comment:
This is actually the first thing in the README that cares what IDE the reader is using. So you should at least mention VS Code rather than "the testing tab".

But it's not ideal if the only way to run tests is via VS Code. You always want the tests to run successfully from command line. In module 7 we will want a pipeline to automatically run the tests. So you should document that (e.g. `poetry run pytest` or `poetry run pytest a_folder_name`)

Also, selenium tests require having a browser and webdriver installed (e.g. Firefox and a suitable version of geckodriver on your PATH). This should be documented in the README to match what your test needs.